lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    cell_start
+  end
+
+  def reset!
+    cell_start
+  end
+
+  def cell_start
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts div_line
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts div_line
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def div_line
+    "-----------"
+  end
+
+  def position(index)
+    @cells[index.to_i - 1]
+  end
+
+  def full?
+    @cells.any? { |cell| cell == " "} ? false : true
+  end
```

Comment:
Use the implicit boolean return value of the expression instead of adding the ternary. Reverse (or negate) the boolean with the `!` (bang) operator.

`!@cells.any? { |cell| cell == " "}`